### ISSUE TITLE
Does the multi-class segmentation model support overlapping classes in the segmented image (output of the network)?

### ISSUE THREAD

**Issue by Fresh-detergent** [qubvel/segmentation_models.pytorch#544]
Hello Segmentation Models developers,

I was wondering about the following: **does the multi-class segmentation model support overlapping classes in the segmented image (output of the network)?** 

Take the following as an example: suppose I have an input image of a car and a building. Suppose that I'm detecting both cars and buildings, but that the car overlaps with the building in the input image (it covers up a part of the building). Nevertheless, I'd like to detect both the car and the building in their entirety. You can assume that I have the annotations for both the car and the building in their entirety.

Is this something I can do by default by using the multi-class segmentation model, or will I need to do something else?

**Comment by qubvel** [comment]
Hi, @Fresh-detergent 
Case with overlapping classes is called 'multilabel'. In general, you build it as the usual multiclass model, however for the last layer use sigmoid instead of softmax (and of course, correct loss function should be used).

**Comment by Fresh-detergent** [comment]
@qubvel great. Are there any pre-existing examples for this use case that I can take a look at?
